File: HarvardX-CS50-project01-Mario(less)/C/mario-less.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    char symbol = '#';
    char offset_symbol = ' ';
    
    do
    {
        height = get_int("Height: ");
    }
    while(height < 1 || height > 8);
    
    
    for(int i = 0; i < height; i++)
    {
        for(int m = height-1; m > i; m--)
        {
            printf("%c", offset_symbol);
        }
        for(int k = 0; k <(i+1); k++)
        {
           
            printf("%c",symbol);
        }
        printf("\n");
    }
    
    
    
}

File: HarvardX-CS50-project01-Mario(less)/Python/mario-less.py
height = 0
symbol = '#'
offset_symbol = ' '

while True:
    height = input("Height: ")

    if height.isdigit():
        height = int(height)

        if height > 0 and height < 9:
            break

for i in range(height):
    for m in range(height-1, i, -1):
        print(f"{offset_symbol}", end="")
    for k in range(i + 1):
        print(f"{symbol}", end="")
    print()